@navikt/core/react/src/modal/Modal.tsx

```diff
@@ -79,111 +94,153 @@ type ModalLifecycle = {
  *
  * @example
  * ```jsx
+ * const ref = useRef<HTMLDialogElement>(null);
```

Comment:
```suggestion
* State-change with `useRef`
 * ```jsx
 * const ref = useRef<HTMLDialogElement>(null);
```